src/test/java/rx/internal/operators/OperatorIgnoreElementsTest.java

```diff
@@ -92,7 +92,7 @@ public void call(Integer t) {
                     }
                 })
                 //
-                .ignoreElements()
+                .<Integer>ignoreElements()
```

Comment:
I just realized we don't have a guidance on source compatibility explicitly. We only discussed backward compatibility in #1917. But I'm supposed backward compatibility should include both binary and source backward compatibility.